core/src/main/java/net/opentsdb/query/BaseQueryContext.java

```diff
@@ -337,18 +337,36 @@ protected void log(final LogLevel level, final QueryNode node, final String log)
       return;
     }
     
+    final StringBuilder buf = new StringBuilder(15 + 6 + 32 + log.length())
```

Comment:
These numbers in a vacuum are magical. I know they're meaningful, but, someday, using named constants instead could help with intelligibility.

Reply:
Added a note and fixed them. Just estimates of the strings we'll be adding so we pre-build the buffer.